src/components/node/news_story/index.tsx

```diff
@@ -0,0 +1,89 @@
+/** These types/packages will import into all node components. */
+import { NodeMetaInfo } from '@/components/node'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+
+/** These component includes are specific to this component. */
+import { MediaImage } from '@/components/media'
+import { StaffNewsProfile } from '@/components/node/person_profile'
+import { NodeNewsStory } from '@/types/node'
+
+type NodeNewsStoryProps = {
+  node: NodeNewsStory
+  viewMode?: string
+}
+
+/** Full page news story. */
+export const NewsStoryFull = ({ node }: NodeNewsStoryProps) => {
+  /** Type narrowing; if we've managed to end up here with the wrong data, return. */
+  if (node.type !== 'node--news_story') return
+  const dateTime = new Date(node.created).toLocaleDateString('en-us', {
```

Comment:
May want to consider having a DATE_FORMAT object in a constants file that would list multiple date formats. Are there many different date formats throughout the site being used?

Reply:
I will refactor this after merge.

Reply:
I am into this idea. Let's refactor.